qiskit_machine_learning/neural_networks/circuit_qnn.py

```diff
@@ -208,6 +213,10 @@ def _sample(self, input_data: Optional[np.ndarray], weights: Optional[np.ndarray
 
     def _probabilities(self, input_data: Optional[np.ndarray], weights: Optional[np.ndarray]
                        ) -> Union[np.ndarray, SparseArray]:
+
+        if not self._quantum_instance:
```

Comment:
Should be `if self._quantum_instance is not None:` for an explicit test

Reply:
@woodsp-ibm should it be
```python
if self._quantum_instance is None:
     raise QiskitMachineLearningError("Quantum instance required")
     
if self._quantum_instance.is_statevector:
     raise QiskitMachineLearningError("Sampling does not work with statevector simulator!")
...
```
(so we avoid executing the remainder of the function if no quantum instance)

Reply:
From my understanding there is _sample and _probabilities. The _sample method, for sampling, does not work with statevector, but either require a quantum_instance hence the split in the testing.

Reply:
@woodsp-ibm thanks, will address this shortly